internal/binder/function/funcs_analytic.go

```diff
@@ -261,10 +261,9 @@ func registerGlobalAggFunc() {
 				return fmt.Errorf("when arg is not a bool but got %v", args[len(args)-2]), false
 			}
 			if v1 == nil && v2 == nil {
-				if args[0] == nil || !validData {
-					return 0, true
+				if !validData {
+					return 0, false
```

Comment:
Shall we return the latest agg value instead of 0 value? Not only this place, all other places have the same problem